packages/components/src/components/filter-type-group/filter-accordion/filter-accordion.tsx

```diff
@@ -0,0 +1,85 @@
+import { Component, h, Prop, State, Event, EventEmitter, Element } from '@stencil/core';
+import { getInitiallySelectedItems } from '../utils';
+
+@Component({
+  tag: 'ifx-filter-accordion',
+  styleUrl: 'filter-accordion.scss',
+  shadow: true,
+})
+export class FilterAccordion {
+  @Element() private el: HTMLElement;
+  @State() expanded: boolean = false;
+  @State() showMore = false;
+  @State() count: number = 0;
+  @State() totalItems = 0;
+  @Prop() filterGroupName = "";
+
+  @Event() ifxFilterAccordionChange: EventEmitter; // Add this line
+
+  componentWillLoad() {
+    this.el.addEventListener('ifxListUpdate', this.handleCheckedChange);
+  }
+
+  componentDidLoad() {
+    // Dispatch the ifxListUpdate event after the component has been fully loaded
+    const selectedItems = getInitiallySelectedItems(this.el);
+    this.count = selectedItems.length;
```

Comment:
This Prop should not be updated inside `componentDidLoad`. It triggers an error warning in the console. It should be updated inside `componentWillLoad()` lifecycle.

Reply:
I need to update it in ComponentDidLoad(), because access to the DOM is required to determine the selected items.
The update to this.count depends on the selected items that are obtained from the DOM.

Reply:
I also dont get any warnings. Can you send a screenshot here as well?